tests/publish.rs

```diff
@@ -12,7 +13,7 @@ fn publish_module_as_user_correct() {
         let res = MoveModule::publish_module(
             // Just for now - as Move module address account is 0xCAFE, we need to sign it the same
             // address. But in tests, AccountId is u64, so we need to convert it (0xCAFE -> 0xFECA000000000000 - endian welcome)
-            RuntimeOrigin::signed(0xFECA000000000000),
+            RuntimeOrigin::signed(AccountId32::new([1u8; 32])),
```

Comment:
Does this test pass when we call `publish_module` signed from a different account than the account specified in the `Move.toml` file?
I'd expect that Move will reject this as the address where the module should be published is not aligned with the caller.

Reply:
Updated to use [1u8; 32] fixture